lib/CL/devices/builtin_kernels.cc

```diff
@@ -14,10 +14,11 @@
 #define WRITE_BUF                                                              \
```

Comment:
This file was accidentally made as a C++ file and it's the only file that is currently C++ in the core runtime, forcing the need of C++ and STL to low end edge devices that want to launch DBKs. Can you please convert this also to C99?

Reply:
converted & moved to `lib/CL/pocl_builtin_kernels.c`